### DISCUSSION TITLE
ScriptReference/Networking.DownloadHandlerTexture-texture

### DISCUSSION THREAD

**Discussion by docscomments** [Developer-Notes-Extension/Unity-Notes#346] [Notes]
# ScriptReference/Networking.DownloadHandlerTexture-texture

https://docs.unity3d.com/ScriptReference/Networking.DownloadHandlerTexture-texture.html

**Comment by Henry00IS** [comment]
The texture returned will be read-only. If you wish to generate mipmaps at runtime, you must copy it to a new texture with "mipChain" set to true. For example:
```csharp
var newTexture = new Texture2D(texture.width, texture.height, texture.format, true); // enable mipmaps on a new texture.  
newTexture.SetPixels32(texture.GetPixels32()); // copy the pixels to the new texture.  
newTexture.Apply(); // generate the mipmaps.
```
